napari/layers/labels/labels.py

```diff
@@ -340,6 +370,37 @@ def selected_label(self, selected_label):
         self._selected_color = self.get_color(selected_label)
         self.events.selected_label()
 
+    @property
+    def color_mode(self):
+        """Color mode string"""
+        if hasattr(self, '_color_mode'):
```

Comment:
We generally have initialized the layer property with the default value, so in the init I'd have `self._color_mode = ColorMode('direct')` and then you wouldn't need the `hasattr` here, for example see how we treat Blending https://github.com/napari/napari/blob/master/napari/layers/base/base.py#L143.